content/blog/2018/02/2018-02-27-browser-usage.adoc

```diff
@@ -0,0 +1,196 @@
+---
+layout: post
+title: "Browser Market Share on Jenkins.io"
+tags:
+- community
+- stats
+- browser
+author: olblak
+---
+
+= Browser Market Share on Jenkins.io
+
+I recently had a look to the google analytics account used for jenkins.io.
```

Comment:
I suggest leading in with some comments that we have been investing a lot of time and effort in bringing more documentation (linked), and other content (link to security advisories) on jenkins.io (all lowercased). To help with further development of jenkins.io, you wanted to understand the browsers people were using to view the site.